vss-extension.json

```diff
@@ -181,7 +196,7 @@
 					{
 						"id": "definition",
 						"name": "Source (repository)",
-						"description": "Repository to pull",
+						"description": "ID of the Repository to pull",
```

Comment:
This is an editable dropdown and ideally should show existing repos. Why do you think ID needs to be mentioned here?
![image](https://user-images.githubusercontent.com/2685029/53158479-96670080-35bb-11e9-93f4-73c128e3b2d3.png)

Reply:
Ok for that my fault ...